### PR TITLE
feat: link to parent collective from connected collective

### PR DESCRIPTION
**Pull request by bolariin**
Resolve https://github.com/opencollective/opencollective/issues/2485

# Description

This PR allows collectives to link back to parent collective.

# Screenshots

with linked collectuve

### CODE REVIEW

File: components/collective-page/graphql/queries.js [hunk 1]
```diff
@@ -53,6 +53,15 @@ export const getCollectivePageQuery = gql`
           all
         }
       }
+      memberOf(role: "CONNECTED_COLLECTIVE") {
```
Comment (Betree) [inline components/collective-page/graphql/queries.js:56]: To make that clearer, I'd suggest aliasing the property (the matching prop will be renamed in `data`):

```suggestion
      connectedTo: memberOf(role: "CONNECTED_COLLECTIVE") {
```

File: components/collective-page/hero/Hero.js [hunk 2]
```diff
@@ -305,6 +324,16 @@ Hero.propTypes = {
     settings: PropTypes.shape({
       tos: PropTypes.string,
     }).isRequired,
+    memberOf: PropTypes.arrayOf(
+      PropTypes.shape({
+        id: PropTypes.number.isRequired,
+        collective: PropTypes.shape({
+          id: PropTypes.number.isRequired,
+          name: PropTypes.string.isRequired,
+          // slug: PropTypes.string.isRequired,
```
Comment (Betree) [inline components/collective-page/hero/Hero.js:333]: Let's remove rather than commenting

```suggestion
```

Comment (bolariin) [reply components/collective-page/hero/Hero.js]: This was a mistake, this shouldn't have been commented out

File: components/collective-page/hero/Hero.js [hunk 3]
```diff
@@ -211,22 +211,41 @@ const Hero = ({ collective, host, isAdmin, onPrimaryColorChange, callsToAction,
...
+                            <Span data-cy="fiscalHostName" color="black.600">
+                              {host.name}
+                            </Span>
+                          </LinkCollective>
+                        ),
+                      }}
+                    />
+                  </Container>
+                  {collective.connectedTo.length !== 0 && (
+                    <Container mx={1} color="#969ba3" my={2}>
```
Comment (Betree) [inline components/collective-page/hero/Hero.js:232]: I just checked https://www.figma.com/file/GF1gQ7fWNfaOAFNwuLeAUl/%5BOC.com%5D-Hot-Fixes?node-id=1081%3A12923 and the margin left here should be `16px`. This should do it: 

```suggestion
                    <Container mx={1} color="#969ba3" my="12px">
```

**Pull request merged**